receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_scraper_test.go

```diff
@@ -193,7 +193,7 @@ func TestScrape(t *testing.T) {
 				expectedMetricCount += 4
 			}
 			if test.expectConnectionsMetric {
-				expectedMetricCount++
+				expectedMetricCount += 2
```

Comment:
Why does this change? Doesn't it break the tests?

Reply:
let's revert and find out

Reply:
```suggestion
				expectedMetricCount++
```

Reply:
that was totally it. Thank you.